src/main/java/edu/cornell/kfs/pdp/document/CuPayeeACHAccountMaintainableImpl.java

```diff
@@ -64,12 +69,25 @@ public boolean answerSplitNodeQuestion(String nodeName) {
         if (StringUtils.equalsIgnoreCase(CUPdpConstants.PAYEE_ACH_ACCOUNT_REQUIRES_PDP_APPROVAL_NODE, nodeName)) {
             return payeeACHAccountMaintenanceRequiresPdpApproval();
         }
-        
         return super.answerSplitNodeQuestion(nodeName);
     }
 
     protected boolean payeeACHAccountMaintenanceRequiresPdpApproval() {
+        try {
+            DocumentService documentService = SpringContext.getBean(DocumentService.class);
+            FinancialSystemMaintenanceDocument document = (FinancialSystemMaintenanceDocument) documentService.getByDocumentHeaderId(this.getDocumentNumber());
+            if (isDocumentInitiatorSystemUser(document)) {
+                return false;
+            }
+        } catch (Exception exception) {
+            LOG.error("payeeACHAccountMaintenanceRequiresPdpApproval " + exception.getMessage(), exception);
+        }
         return StringUtils.equalsIgnoreCase(KFSConstants.MAINTENANCE_EDIT_ACTION, getMaintenanceAction());
     }
 
+    protected boolean isDocumentInitiatorSystemUser(FinancialSystemMaintenanceDocument document) {
+        Person documentInitiator = KimApiServiceLocator.getPersonService().getPerson(document.getDocumentHeader().getWorkflowDocument().getInitiatorPrincipalId());
+        return documentInitiator != null && documentInitiator.getPrincipalName().equalsIgnoreCase(KFSConstants.SYSTEM_USER);
```

Comment:
For better null safety, you could use StringUtils.equalsIgnoreCase() to compare the principal names.